src/nvim/mouse.c

```diff
@@ -1756,18 +1756,25 @@ static win_T *mouse_find_grid_win(int *gridp, int *rowp, int *colp)
     }
   } else if (*gridp == 0) {
     ScreenGrid *grid = ui_comp_mouse_focus(*rowp, *colp);
-    FOR_ALL_WINDOWS_IN_TAB(wp, curtab) {
-      if (&wp->w_grid_alloc != grid) {
-        continue;
-      }
+    if (grid == &pum_grid) {
       *gridp = grid->handle;
-      *rowp -= grid->comp_row + wp->w_grid.row_offset;
-      *colp -= grid->comp_col + wp->w_grid.col_offset;
-      return wp;
+      *rowp -= grid->comp_row + grid->row_offset;
+      *colp -= grid->comp_col + grid->col_offset;
+      // TODO(fredizzimo): Does the popup have a window?
```

Comment:
@zeertzjq

I guess I should change this comment to:
"The popup menu doesn't have a window"

Reply:
I changed the comment